### PR TITLE
finish first draft of frontend

### PR DESCRIPTION
**Pull request by oriooctopus**

### CODE REVIEW

File: client/src/components/QuestionsListItem.jsx [hunk 1]
```diff
@@ -1,12 +1,12 @@
...
 
 const QuestionListItem = props => (
-  <li>
+  <li onClick={props.changeRight} >
     <div>
-      <img src="https://placeholdit.imgix.net/~text?txtsize=9&txt=100%C3%97100&w=100&h=100" />
+      <img className="questionImage" src="https://placeholdit.imgix.net/~text?txtsize=9&txt=100%C3%97100&w=100&h=100" />
       <div>
         <h3 className="username">Oliver Ullman:</h3>
-        <h3 className="questionTitle">Is it okay that I don't shower after a workout?</h3>
```
Comment (Aelgiadi) [inline client/src/components/QuestionsListItem.jsx]: But why????

Comment (msermersheim) [inline client/src/components/QuestionsListItem.jsx:9]: hahaha

File: client/src/index.jsx [hunk 2]
```diff
@@ -1,18 +1,43 @@
...
+    super(props);
+    this.state = {
+      answerDisplayClass: 'invisible',
+      askDisplayClass: 'block'
+    }
+    this.changeRight = this.changeRight.bind(this);
+  }
+
+  changeRight() {
+    console.log(1234);
```
Comment (Aelgiadi) [inline client/src/index.jsx]: Remove the "console.log" on line 20, it would be silly to merge this to master.

**Review by Aelgiadi — CHANGES_REQUESTED** [review]
Over all looks good. Remove the "console.log" and then we can merge it. Good work man!

**Pull request closed without merging**